docs2/code_overview/models.rst

```diff
@@ -109,9 +114,255 @@ Player
     :private-members:
     :inherited-members:
 
-Raid Member
+Wars
+~~~~~
+
+Clan War
+^^^^^^^^
+.. autoclass:: ClanWar()
+    :members:
+    :private-members:
+    :inherited-members:
+
+Clan War Log Entry
+^^^^^^^^^^^^^^^^^^
+.. autoclass:: ClanWarLogEntry()
+    :members:
+    :private-members:
+    :inherited-members:
+
+Clan War League Group
+^^^^^^^^^^^^^^^^^^^^^
+.. autoclass:: ClanWarLeagueGroup()
+    :members:
+    :private-members:
+    :inherited-members:
+
+WarAttack
+^^^^^^^^^^
+.. autoclass:: WarAttack()
+    :members:
+    :private-members:
+
+War League
+^^^^^^^^^^
+.. autoclass:: WarLeague()
+    :members:
+    :private-members:
+
+
+Raids
+~~~~~
+
+Raid Log Entry
+^^^^^^^^^^^^^^
+.. autoclass:: RaidLogEntry()
+    :members:
+    :private-members:
+
+Raid District
+^^^^^^^^^^^^^
+.. autoclass:: RaidDistrict()
+    :members:
+    :private-members:
+
+Raid Attack
 ^^^^^^^^^^^
-.. autoclass:: RaidMember()
+.. autoclass:: RaidAttack()
+    :members:
+    :private-members:
+
+Game Objects
+~~~~~~~~~~~~
+
+Troop
+^^^^^
+.. autoclass:: Troop()
+    :members:
+    :private-members:
+
+
+Hero
+^^^^
+.. autoclass:: Hero()
+    :members:
+    :private-members:
+
+Pet
+^^^
+.. autoclass:: Pet()
+    :members:
+    :private-members:
+
+
+Spell
+^^^^^
+.. autoclass:: Spell()
+    :members:
+    :private-members:
+
+Player Attributes
+~~~~~~~~~~~~~~~~~
+
+Achievement
+^^^^^^^^^^^
+.. autoclass:: Achievement()
+    :members:
+    :private-members:
+
+
+League
+^^^^^^
+.. autoclass:: League()
     :members:
     :private-members:
     :inherited-members:
+
+Legend Statistics
+^^^^^^^^^^^^^^^^^
+.. autoclass:: LegendStatistics()
+    :members:
+    :private-members:
+    :inherited-members:
+
+Label
+^^^^^
+.. autoclass:: Label()
+    :members:
+    :private-members:
+
+Graphics
+~~~~~~~~
+
+Badge
+^^^^^
+.. autoclass:: Badge()
+    :members:
+    :private-members:
+
+Icon
+^^^^
+.. autoclass:: Icon()
+    :members:
+    :private-members:
+
+Time
+~~~~
+
+Timestamp
+^^^^^^^^^
+.. autoclass:: Timestamp()
+    :members:
+    :private-members:
+
+
+TimeDelta
+^^^^^^^^^
+.. autoclass:: TimeDelta()
+    :members:
+    :private-members:
+
+Location
+~~~~~~~~
+.. autoclass:: Location()
+    :members:
+    :private-members:
+
+Enumerations
+~~~~~~~~~~~~
+
+The library provides some enumerations for certain types of strings, as well as orders for troops, spells and
+achievements that are used internally.
+
+Corresponds to a member's in-game role in the clan.
```

Comment:
should be in line 280 shouldn't it? below the class but above the attribute?

Reply:
yes, but we can delete 281 and 282 imo

Reply:
or we don't, but get rid of these "corresponds" sentences